tests/network/libs/cloudinit.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
```

Comment:
repo's python 3.12, why is this needed?

Reply:
No idea, maybe a leftover from old times. I will remove it.

Reply:
I found out why.. It allows the classes to be ordered from top to bottom and not like a script. I.e. refer to a class before it is defined, like any reasonable language.
But it is not worth it, we can do it when that release will support it and not before.

Fixed